lib/logger/utils.ts

```diff
@@ -46,7 +47,74 @@ const contentFields = [
   'yarnLockParsed',
 ];
 
+type ZodShortenedIssue =
+  | null
+  | string
+  | string[]
+  | {
+      [key: string]: ZodShortenedIssue;
+    };
+
+export function prepareZodIssues(input: unknown): ZodShortenedIssue {
+  // istanbul ignore if
+  if (!is.plainObject(input)) {
+    return null;
+  }
+
+  let err: null | string | string[] = null;
+  if (is.array(input._errors, is.string)) {
```

Comment:
do we need unknown because of private `_errors` property? hopefully it won't break in future.

Reply:
It's not private property, it was made in order to show deep errors without conflicting with fields defined in the schema itself. I don't see how this can conflict, so I'm extracting these error strings to the upper level